posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+title: "Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reece-nana
+author_github: https://github.com/reece-nana
+seo-title: Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+blog_description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot off the press link:https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]. Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this beta release, the features that make up the new Jakarta EE 10 Core Profile are made available: 
```

Comment:
I think this opening paragraph (ie everything before the sentence "The 22.0.0.9-beta includes the following beta features (along with all GA features):") is too long and gets into too much detail about Jakarta 10 too early. It also doesn't really say what "Core Profile" means. There's also a fair amount of duplication with having both bulleted lists. And the second bulleted list (and sub-headings throughout the blog post) need to focus more on value instead of feature/functions.

How about:

Open Liberty 22.0.0.9-beta brings you the hot-off-the-press [Jakarta EE 10 Core Profile](https://jakarta.ee/specifications/coreprofile/10/). Jakarta EE 10 will be the first release to provide updates to the specifications since Java EE 8 in 2017. Jakarta EE 10 Core Profile is a subset of the specifications. The Core Profile is specifically aimed at lightweight runtimes, like Open Liberty, that are optimised for running cloud-native Java microservices. While the whole Jakarta EE 10 release is just around the corner, Open Liberty 22.0.0.9-beta provides a preview of the Core Profile subset of features for you to start exploring now.

As well as Jakarta EE 10 Core Profile, we've included Jakarta EE 10 Expression Language 5.0 in this beta, which provides a number of new features and specification clarifications.

But it’s not just Jakarta EE features that are making their debut in this release. MicroProfile OpenAPI 3.1 provides the ability to specify optional authentication for operations and multiple annotation enhancements. Password Utilities 1.1 is introduced in this beta, and with it the ability to no longer forcefully federate stand-alone user registries.

The [Open Liberty](https://blogs-draft-openlibertyio.ng.bluemix.net/) 22.0.0.9-beta includes the following beta features (along with all GA features):

* Jakarta EE 10 Core Profile:

** [Support for multipart/form-data content type (RESTful Web Services 3.1)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#restful)

** [Enhanced JSON string parsing and new serialization/deserialization options (JSON Processing 2.1 and JSON Binding 3.0)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#json)

** [KEY BENEFIT OF RELEASE HERE? (CDI 4.0)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#cdi)

*[Access any annotations present on a particular method (Expression Language 5.0)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#expression)

* [Document APIs where authentication is optional or complex (MicroProfile OpenAPI 3.1)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#microprofile)

* [KEY BENEFIT OF RELEASE HERE? (Password Utilities 1.1)](https://blogs-draft-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html#password)
"

Then make all the sub-headings match the links in the list above.